frontend/src/components/regions/CardDetailsModal/Presentation.js

```diff
@@ -62,66 +64,71 @@ const CardBasicDetails = ({ card }) => {
-          <TagChips tagNames={card.tagNames} />
-          <FlavourChips flavourNames={card.flavourNames} />
-          <StoryPoints storyPoints={card.storyPoints} />
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Typography variant="subtitle2">Assignees:</Typography>
-            <UsersTable userNames={card.assigneeNames} userIds={card.assignees} />
-            <Typography variant="subtitle2">Reviewers:</Typography>
-            <UsersTable userNames={card.reviewerNames} userIds={card.reviewers} />
-          </Paper>
-        </Grid>
-        <Grid item xs={12} sm={12} md={12}>
-          <Paper className={classes.sectionPaper} variant="outlined">
-            <Table>
-              <TableBody>
-                <TableRow>
-                  <TableCell>Due Time</TableCell>
-                  <TableCell>{dueTime}</TableCell>
-                </TableRow>
-                {card.startTime && (
-                  <TableRow>
-                    <TableCell>Start Time</TableCell>
-                    <TableCell>{startTime}</TableCell>
-                  </TableRow>
-                )}
-
-                {card.reviewRequestTime && (
-                  <TableRow>
-                    <TableCell>Review Request Time</TableCell>
-                    <TableCell>{reviewRequestTime}</TableCell>
-                  </TableRow>
-                )}
-                {card.completeTime && (
-                  <TableRow>
-                    <TableCell>Complete Time </TableCell>
-                    <TableCell>{completeTime}</TableCell>
-                  </TableRow>
-                )}
-              </TableBody>
-            </Table>
-          </Paper>
-        </Grid>  
-      </Grid> 
+      <Typography variant="h5">
+        {card.contentTypeNice}: {card.title}
+      </Typography>
+      <TagChips tagNames={card.tagNames} />
+      <FlavourChips flavourNames={card.flavourNames} />
+      <StoryPoints storyPoints={card.storyPoints} />
+
+      <Paper className={classes.sectionPaper} variant="outlined">
+        <Typography variant="subtitle2">Assignees:</Typography>
+        <UsersTable userNames={card.assigneeNames} userIds={card.assignees} />
+        <Typography variant="subtitle2">Reviewers:</Typography>
+        <UsersTable userNames={card.reviewerNames} userIds={card.reviewers} />
+      </Paper>
+
+      <Paper className={classes.sectionPaper} variant="outlined">
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell>Due Time</TableCell>
+              <TableCell>
+                {dueTime}
+              </TableCell>
+              {
+                authUser.email === card.assigneeNames[0] && !dueTime && 
+                Object.keys(authUser.permissions.teams)
+                .map((key) => authUser.permissions.teams[key].permissions[0])
+                .includes("MANAGE_CARDS") ? (
+                  <TableCell>
```

Comment:
This row of the table sometimes has 3 cells and sometimes has 2.

All other rows only have 2 cells. Why do we want the button to be so far to the right of the screen? Why make the table inconsistent?

If you just put the button in the same cell as the due time text then what would go wrong?

eg:

```
<TableCell>
                {dueTime} {canSetDueTime({card,authUser}) && (<Button variant="outlined">Set Date</Button>)}
</TableCell>
```